test/Mvc.Api.Analyzers.Test/TestFiles/SymbolApiResponseMetadataProviderTest/TryGetActualResponseMetadataTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Mvc.Api.Analyzers.TestFiles.SymbolApiResponseMetadataProviderTest
+{
+    public class TryGetActualResponseMetadataController : ControllerBase
+    {
+        public async Task<ActionResult<IEnumerable<TryGetActualResponseMetadataModel>>> ActionWithActionResultOfTReturningOkResult()
+        {
+            await Task.Yield();
+            var models = new List<TryGetActualResponseMetadataModel>();
+
+            return Ok(models);
+        }
+
+        public async Task<ActionResult<IEnumerable<TryGetActualResponseMetadataModel>>> ActionWithActionResultOfTReturningModel()
+        {
+            await Task.Yield();
```

Comment:
you can avoid an actual context switch if you use `await Task.CompletedTask` instead.